src/app/utils/formatAge.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Formats a given date string into a friendly age format.
+ * The friendly age format includes hours, minutes, and seconds.
+ * @param dateString The date string to format.
+ * @returns The formatted age string.
+ */
+export function formatAge(dateString: string): string {
```

Comment:
there is a library already included that does this formatting I think, no need to do it by coding a function again, but I'm not sure.

Reply:
if you add functions like this, you need a unit test to cover as well

Reply:
Can't find a library that directly does this formatting. Also added a unit test for this.